jdbc-handler/src/main/java/org/apache/hive/storage/jdbc/dao/GenericJdbcDatabaseAccessor.java

```diff
@@ -228,6 +233,13 @@ protected Properties getConnectionPoolProperties(Configuration conf) {
       }
     }
 
+    // handle password
+    Credentials credentials = UserGroupInformation.getCurrentUser().getCredentials();
+    if (credentials.getSecretKey(DBCP_PWD) != null) {
+      LOGGER.info("found token in credentials");
```

Comment:
Ok, how about making it more easy for user to understand ? Change message to "Obtained db password from credential store" ?